pkg/agent/baker.go

```diff
@@ -623,6 +623,9 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"IsAzureCNIOverlayFeatureEnabled": func() bool {
 			return cs.Properties.OrchestratorProfile.KubernetesConfig.IsUsingNetworkPluginMode("overlay")
 		},
+		"EbpfDataplane": func() bool {
```

Comment:
seems like we should change the func name to `EbpfDataplaneCilium` here to reflect what we're actually checking for?

Reply:
The only valid value for `EbpfDataplane` is Cilium, so renamed the function to `CiliumDataplaneEnabled`.